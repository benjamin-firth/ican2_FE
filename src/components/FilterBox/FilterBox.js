import React from 'react';
import './FilterBox.scss';

const FilterBox = () => {
  window.scrollTo(0, 0);

  return (
    <section className='filter-container'>
      <h5>this is the filter box</h5>
    </section>
  );
}

export default FilterBox;
