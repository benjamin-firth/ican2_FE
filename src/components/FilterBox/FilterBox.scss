@import '../../variables.scss';

// * {
//   border: 1px solid blue;
// }

.filter-container {
  position: fixed;
  background-color: $primarycolor;
  width: 100%;
  text-align: center;
  margin-top: 1.4vw;
  z-index: 0;


  h5 {
    font-family: $secondaryfont;
    color: $secondarycolor;
    font-size: 2vw;
  }
}
