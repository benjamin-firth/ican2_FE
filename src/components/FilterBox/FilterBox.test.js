import React from 'react';
import { shallow } from 'enzyme';
import FilterBox from './FilterBox';

describe('FilterBox', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow( <FilterBox />)
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

});
