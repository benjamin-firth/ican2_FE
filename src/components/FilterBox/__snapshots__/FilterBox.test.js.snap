// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilterBox should match the snapshot 1`] = `
<section
  className="filter-container"
>
  <h5>
    this is the filter box
  </h5>
</section>
`;
